platforms/hyperledger-besu/configuration/roles/delete/certificates/ambassador/tasks/nested_main.yaml

```diff
@@ -0,0 +1,16 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+
+# This role generates certificates for rootca and ambassador
+# and places them in vault. Certificates are created using openssl
+---
+# #delete ambassador tls certificates created by cert-manager 
```

Comment:
This can be renamed to `# Delete ambassador tls certificates created by cert-manager)` (Single #)

Reply:
fixed in pr #2187 as unwanted files got added while squashing